### ISSUE TITLE
Replace Redis with RabbitMQ as celery backend

### ISSUE THREAD

**Issue by giocalitri** [mitodl/odl-video-service#170]
This will allow an easy deployment for EC2 and we will not have to deal with external services that might cause crashes of our application in case of maximum number of connections.

**Comment by pdpinch** [comment]
Is this necessary for #54 ?

**Comment by giocalitri** [comment]
No, this is because this app uses way more connections to redis than other applications and we can easily run out of them if we scale the number of celery workers.
I talked with Tobias and he told me that we have reliable formulas to deploy our own RabbitMQ  cluster. Then I spoke with Nathan and we agreed that RabbitMQ is a better choice as broker in the long term for Celery.
